vmaas/tests/conftest.py

```diff
@@ -15,11 +16,45 @@ def pytest_configure(config):
     config.addinivalue_line('markers', 'smoke: mark a test as a smoke test.')
 
 
+@pytest.fixture(name='make_conn')
+def _make_conn():
```

Comment:
Why not simply
```python
@pytest.fixture
def make_conn():
    def _make_conn():
    ...
   return _make_conn
```
?
Also yield fixture is not necessary here (``return`` is enough) since we are not doing any cleanup once the fixture goes out of scope.

Reply:
Okay, I will simplify this

Reply:
Looking at it again, I don't think you need the ``make_conn`` fixture at all. More clean would be to move the ``rest_api`` function to vmaas/rest/tools.py, import it to conftest.py and use it in the ``rest_api`` fixture and import it to vmaas/tests/test_updates.py and use it as a normal function in tests.

Reply:
Good idea!